csrc/fusion_profiler.cpp

```diff
@@ -574,15 +574,19 @@ void cupti_buffer_requested(
     uint8_t** ppBuffer,
     size_t* pSize,
     size_t* pMaxNumRecords) {
-  uint8_t* pBuffer = FusionProfiler::cuptiBufferPtr();
```

Comment:
Is the global [cupti_buffer_](https://github.com/NVIDIA/Fuser/blob/main/csrc/fusion_profiler.h#L296-L297) needed anymore then?

Reply:
Thanks for catching that, I have removed it.